writing-content/image-embeds.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Images and Embeds'
+description: 'Add image, video, and other HTML elements'
+---
+
+<img
+  className="rounded-md"
+  src="https://mintlify-assets.b-cdn.net/bigbend.jpg"
+/>
+
+## Image
+
+### Using Markdown
+
+The [markdown syntax](https://www.markdownguide.org/basic-syntax/#images) lets you add images using the following code
+
+```md
+![title](/path/image.jpg)
+```
+
+Note that the image file size must be less than 5MB. Otherwise, we recommend hosting on a service like [Cloudinary](https://cloudinary.com/) or [S3](https://aws.amazon.com/s3/). You can then use that URL and embed. 
+
+### Using Embeds
+
+To get more customizability with images, you can also use [embeds](/writing-content/embed) to add images
+
+```html
+<img height="200" src="/path/image.jpg" />
+```
+
+<Tip>
+
+Although not required, we recommend adding the `alt` and `title` attributes to images for better SEO and accessability. Learn more at [image SEO](https://yoast.com/image-seo-alt-tag-and-title-tag-optimization/).
+
+</Tip>
+
+### Related
+
+For more information, we recommend the following sections:
+
+<Card title="Frame Component Reference" icon="frame" href="/components/frame">
+  Read the reference for the Frame component
+</Card>
+
+## Videos and other HTML elements
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/4KzFe50RQkQ"
+  title="YouTube video player"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+<br />
+
+<Tip>
+
+Mintlify supports [HTML tags in Markdown](https://www.markdownguide.org/basic-syntax/#html). This is helpful if you prefer HTML tags to Markdown syntax, and lets you create documentation with infinite flexibility.
+
+</Tip>
+
+Here are some common HTML tags that are used for documentation
```

Comment:
is there suposed to be smth under this?

Reply:
I took it from the original embeds page, where it just had info about iframes, images, and line breaks under it. i removed the line break

Reply:
maybe remove bc it doesnt make sense